### DISCUSSION TITLE
Prompt is not visible in production server.

### DISCUSSION THREAD

**Discussion by tswapy** [google/model-viewer#4059] [Q&A]
Prompt is visible in local server but not in production.

**Comment by tswapy** [comment]
Is there anyone else facing this issue?

**Comment by elalish** [comment]
We're going to need a lot more details to be able to help. That's why you should never delete the issue template.

**Reply by tswapy** [reply]
Sorry for that. Issue is resolved after more studies. Scene Viewer provides google's default prompts which is cool. When i was working in local server it was using Scene Viewer but when app was in production it was using webXR. WebXR doesn't provide any prompt. Earlier it was `<model-viewer ar-modes="webxr scene-viewer" .... ></model-viewer>` then i changed it to `<model-viewer ar-modes="scene-viewer webxr" .... ></model-viewer>`. So the sequence was important. Issue is resolved.
